### ISSUE TITLE
Question About the Test Dataset

### ISSUE THREAD

**Issue by ShengYu724** [NVlabs/LSM#3]
Thank you for your excellent work!
I’m curious about the specific 40 unseen scenes from ScanNet that you mentioned in your article. Could you elaborate? The number of original test scene of ScanNet is about 100.

**Comment by jzh15** [comment]
In fact, our test set is derived from a portion of the original ScanNet training set, which was then filtered. This is because some scenes in ScanNet contain invalid pose files, rendering the corresponding scenes unusable. The complete test code is expected to be made public before March 15th.
